blueprints/frost-component/files/__root__/__path__/__name__.js

```diff
@@ -1,13 +1,34 @@
 import Ember from 'ember'
-import FrostEvents from 'ember-frost-core/mixins/frost-events'
-import {
-  PropTypes
-} from 'ember-prop-types'
+const { Component } = Ember
+import PropTypesMixin, { PropTypes } from 'ember-prop-types'
+import layout from '<%= templatePath %>'
 
-const {
-  Component
-} = Ember
+const <%= classifiedModuleName %> = Component.extend(PropTypesMixin, {
+
+  // == Dependencies ==========================================================
+
+  // == Component properties ==================================================
+
+  layout,
+
+  // == State properties ======================================================
+
+  propTypes: {
+    hook: PropTypes.string
+  },
+
+  getDefaultProps () {
+    return {
+      hook: '<%= dasherizedModuleName %>'
+    }
+  }
+
+  // == Functions =============================================================
+
+  // == Events ================================================================
+
+  // == Actions ===============================================================
 
-<%= importTemplate %>
-export default Component.extend(FrostEvents, {<%= contents %>
 })
+
+export default <%= classifiedModuleName %>
```

Comment:
Helpers use it a fair amount - basically when the property you require is so intrinsically tied to the name of the component that it doesn't really make sense to "duplicate" it in a property.

Something like `{{play-sound 'foghorn'}}` instead of `{{play-sound sound='foghorn'}}`